net_util/src/queue_pair.rs

```diff
@@ -193,7 +197,7 @@ impl RxVirtio {
                 .ok_or(NetQueuePairError::DescriptorInvalidHeader)?;
             let mut next_desc = Some(desc);
 
-            let mut iovecs = Vec::new();
+            let mut iovecs = self.iovecs.borrow();
```

Comment:
Thanks for your contribution - but aybe we could just use `smallvec` instead - like we do elsewhere in the codebase?

Reply:
To use SmallVec efficiently we need to know beforehand the optimum size, otherwise it will allocate and deallocate. Not sure how feasible this is with the code here.

Reply:
Adding some debugging (e.g. adding `info!("iocount {}", iovecs.len())`) and running various workloads shows that the vector never exceeds length 4 (and that only occurs when copying a very large file). I think a smallvec with reserved capacity of 4 is the best solution.